lib/shared-method.js

```diff
@@ -109,6 +110,13 @@ function SharedMethod(fn, name, sc, options) {
   }
   this.returns.forEach(normalizeArgumentDescriptor);
 
+  var firstReturns = this.returns[0];
+  var streamTypes = ['ReadableStream', 'WriteableStream', 'DuplexStream'];
+
+  if (firstReturns && firstReturns.type && streamTypes.indexOf(firstReturns.type) > -1) {
+    this.stream = firstReturns;
```

Comment:
I'd like to eventually add support for streams to input arguments too. Would you mind using a different property name to distinguish between "accepts" and "returns" streams? E.g. "streamResult" or "streamReturn"